types/mongodb/test/collection/updateX.ts

```diff
@@ -213,7 +213,7 @@ async function run() {
     buildUpdateQuery({ $push: { maybeFruitTags: 'apple' } });
     buildUpdateQuery({
         $push: {
-            subInterfaceArray: { field1: 'foo' },
+            subInterfaceArray: { _id: new ObjectId(), field1: 'foo' },
```

Comment:
This previously inserted an object without `_id` into the array, even if that was illegal according to the `TSchema` type 😱